pkg/codegen/testing/test/testdata/simple-enum-schema/python/pulumi_plant/_enums.py

```diff
@@ -32,6 +32,7 @@ class CloudAuditOptionsLogName(str, Enum):
     """
     What if triple quotes \"\"\" are used in the description
     """
+    CLOUD_AUDIT_OPTIONS_LOG_NAME_N_O_NAME = "_NO_NAME"
```

Comment:
N_O doesn't look right?

Reply:
It doesn't look right, but it is orthogonal to this feature. I'd like to keep this PR focused. I'll open a separate issue to address this.